libs/gl-plugin/src/lib.rs

```diff
@@ -154,6 +155,36 @@ async fn on_peer_connected(plugin: Plugin, v: serde_json::Value) -> Result<serde
     Ok(json!({"result": "continue"}))
 }
 
+async fn on_openchannel(plugin: Plugin, v: serde_json::Value) -> Result<serde_json::Value> {
+    debug!("Received an openchannel request: {:?}", v);
+    let mut rpc = cln_rpc::ClnRpc::new(plugin.configuration().rpc_file).await?;
+    
+    let req = cln_rpc::model::requests::ListdatastoreRequest{
+        key: Some(vec![
+            "glconf".to_string(),
+            "channel".to_string(),
+            "close_to_addr".to_string(),
+        ])
+    };
+
+    let res = rpc.call_typed(req).await;
+    debug!("openchannel response: {:?}", res);
```

Comment:
This is really a `ListdatastoreRequest`, not an `openchannel` response

Reply:
Changed